docs/running-on-kubernetes.md

```diff
@@ -56,14 +56,13 @@ be run in a container runtime environment that Kubernetes supports. Docker is a
 frequently used with Kubernetes. With Spark 2.3, there are Dockerfiles provided in the runnable distribution that can be customized
 and built for your usage.
```

Comment:
I think it worths having a dedicated sub-section here on how to use custom images, e.g., named `Customizing Container Images`.

Reply:
Separate change. I don't even know what you'd write there. The whole "custom image" thing needs to be properly specified first - what exactly is the contract between the submission code and the images, for example.

Reply:
I agree that we don't have a solid story around customizing images here. But I do think that we need something clearly telling people that we do support using custom images if they want to and the properties they should use to configure custom images. It just doesn't need to be opinionated on things like the contact you mentioned.

Reply:
Still, that sounds like something that should be added in a separate change. I'm not changing the customizability of images in this change.

And not having a contract means people will have no idea of how to customize images, so you can't even write proper documentation for that.

Reply:
OK, I am fine with adding this in a future change.